### PR TITLE
[cs] Add option to ignore C# attributes (annotations)

### PR DESCRIPTION
**Pull request by maikelsteneker**
## Describe the PR

For Java, it's possible to ignore annotations. This PR implements the same for C# attributes (which is the closest equivalent to annotations in that language).

You can enable this option on the command line with `--ignore-annotations` or using the CPD GUI.

## Ready?

- [x] Added unit tests for fixed bug/feature
- [x] Passing all unit tests
- [x] Complete build `./mvnw clean verify` passes (checked automatically by github actions)
- [x] Added (in-code) documentation (if needed)

### CODE REVIEW

File: pmd-cs/src/main/java/net/sourceforge/pmd/cpd/CsTokenizer.java [hunk 1]
```diff
@@ -20,26 +20,24 @@ public class CsTokenizer extends AntlrTokenizer {
...
      * @see #OPTION_IGNORE_LITERAL_SEQUENCES
+     * @see #IGNORE_ANNOTATIONS
      */
     public void setProperties(Properties properties) {
-        ignoreUsings = Boolean.parseBoolean(properties.getProperty(IGNORE_USINGS, Boolean.FALSE.toString()));
-        ignoreLiteralSequences = Boolean.parseBoolean(properties.getProperty(OPTION_IGNORE_LITERAL_SEQUENCES,
-            Boolean.FALSE.toString()));
+        ignoreUsings = getBooleanProperty(properties, IGNORE_USINGS);
+        ignoreLiteralSequences = getBooleanProperty(properties, OPTION_IGNORE_LITERAL_SEQUENCES);
+        ignoreAttributes = getBooleanProperty(properties, IGNORE_ANNOTATIONS);
```
Comment (adangel) [inline pmd-cs/src/main/java/net/sourceforge/pmd/cpd/CsTokenizer.java:36]: Ok, so you are using here the common IGNORE_ANNOTATIONS option. That means, this is also supported at the command line via `--ignore-annotations`.

Could you please also update the doc for this then? Currently it says, only Java supports this option. We would also need to mention in one sentence, that the "annotations" are "attributes" in C#.
Here's the doc page: https://pmd.github.io/latest/pmd_userdocs_cpd.html#cli-options-reference and https://github.com/pmd/pmd/blob/master/docs/pages/pmd/userdocs/cpd/cpd.md

**Comment by pmd-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="true">Compared to master:<br />
This changeset changes 0 violations,<br />
introduces 0 new violations, 0 new errors and 0 new configuration errors,<br />
removes 0 violations, 0 errors and 0 configuration errors.<br />
<a href="https://chunk.io/pmd/87a11bd0f3554ef0a845d1f7bcb64d04/diff1/index.html">Full report</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by adangel — COMMENTED** [review]
Thanks!

This looks almost ready. Could you please also update the CLI doc for CPD? (see comment).

**Comment by maikelsteneker** [comment]
Thanks for the useful comment! I have updated the documentation.

**Review by adangel — APPROVED** [review]
Thanks!

**Pull request merged**